crates/cli/tests/cli_run.rs

```diff
@@ -821,6 +821,7 @@ mod cli_run {
     #[test]
     #[cfg_attr(windows, ignore)]
     #[serial(cli_platform)]
+    #[ignore = "Disabled until https://github.com/roc-lang/basic-cli/pull/183 lands"]
```

Comment:
I think this can be unignored now.

Reply:
I think we still need a basic-cli release because it's loaded by URL. @Anton-4 Could we make a small release with that fix?

Reply:
I'll do a minor release of basic-cli today because we need this PR to fix things for the major release (https://github.com/roc-lang/basic-cli/issues/187).

Reply:
basic-cli 0.9.1 is out, #6668 has all links updated.

Reply:
Awesome, thanks!

Reply:
Restored the test [here](https://github.com/roc-lang/roc/pull/6658/commits/6fffaa6ddae0ecaa5adb76160cc10d55479e53f0)